nimbus/src/nimbus.idl

```diff
@@ -19,8 +19,14 @@ dictionary EnrolledExperiment {
     string user_facing_description;
     string branch_slug;
     string enrollment_id;
+    sequence<Branch> branches;
 };
 
+dictionary Branch {
+    string slug;
+    u32 ratio;
+}
```

Comment:
This is missing a semi-colon and is the root cause of the error you are seeing.  See here for more info: https://github.com/mozilla/uniffi-rs/issues/337